libs/output-mapping/src/Writer/Helper/SliceCommandBuilder.php

```diff
@@ -41,4 +43,12 @@ public static function createProcess(
             timeout: 7200.0,
         );
     }
+
+    private static function resolveDataType(string $dataType): string
```

Comment:
já ty statický metody nemám rád :)

Reply:
ani privatní statický? mě je to celkem buřt tak to přehodím

Reply:
jo tak nepřehodím, protože ta "createProcess" je taky statická